tool/converters/verilog_json.sh

```diff
@@ -19,7 +19,9 @@ if !(test 1 -eq $#); then
 fi;
 
 yosys_bin=/oss-cad-suite/bin/yosys
-module=`basename $1 .v`
+sv2v=/usr/local/bin/sv2v
```

Comment:
this is nice for the pre-generated HTMLs, but what about in the confapp schematics?

Reply:
Weird.  Those seem to be working.  I need to check why.  Also, that reminds me of a TODO:  d3-hwschematic had some default options passed to yosys and I don't think I followed them.